sesman/libscp/libscp_session.c

```diff
@@ -416,6 +416,21 @@ scp_session_set_addr(struct SCP_SESSION *s, int type, const void *addr)
 }
 
 /*******************************************************************/
+int
+scp_session_set_guid(struct SCP_SESSION *s, const tui8 *guid)
```

Comment:
If tui8 a useful type? uint8_t is a standard type that is also unsigned 8-bit integer.